index.js

```diff
@@ -0,0 +1,132 @@
+const inquirer = require("inquirer");
+const Manager = require("./src/Manager");
+const Engineer = require("./src/Engineer");
+const Intern = require("./src/Intern");
+const generateHTML = require("./src/generateHTML");
+const fs = require("fs");
```

Comment:
try to import any external modules and packages (code you haven't written) before importing your own utils/classes for clarity :-)